<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>MainWindow</class>
 <widget class="QMainWindow" name="MainWindow">
  <property name="windowModality">
   <enum>Qt::ApplicationModal</enum>
  </property>
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>813</width>
    <height>592</height>
   </rect>
  </property>
  <property name="windowTitle">
   <string>Парсер страниц</string>
  </property>
  <widget class="QWidget" name="centralwidget">
   <layout class="QGridLayout" name="gridLayout_2">
    <item row="1" column="0">
     <widget class="QFrame" name="frame">
      <property name="sizePolicy">
       <sizepolicy hsizetype="Minimum" vsizetype="Minimum">
        <horstretch>0</horstretch>
        <verstretch>1</verstretch>
       </sizepolicy>
      </property>
      <property name="frameShape">
       <enum>QFrame::StyledPanel</enum>
      </property>
      <property name="frameShadow">
       <enum>QFrame::Raised</enum>
      </property>
      <layout class="QGridLayout" name="gridLayout">
       <property name="margin">
        <number>0</number>
       </property>
       <item row="0" column="1">
        <widget class="QSpinBox" name="threads_sb">
         <property name="minimum">
          <number>1</number>
         </property>
         <property name="value">
          <number>3</number>
         </property>
        </widget>
       </item>
       <item row="0" column="0">
        <widget class="QLabel" name="label">
         <property name="text">
          <string>Потоков</string>
         </property>
         <property name="buddy">
          <cstring>threads_sb</cstring>
         </property>
        </widget>
       </item>
      </layout>
     </widget>
    </item>
    <item row="1" column="1">
     <widget class="QDialogButtonBox" name="buttonBox">
      <property name="standardButtons">
       <set>QDialogButtonBox::Cancel|QDialogButtonBox::Save</set>
      </property>
     </widget>
    </item>
    <item row="0" column="0" colspan="2">
     <widget class="QSplitter" name="splitter">
      <property name="sizePolicy">
       <sizepolicy hsizetype="Expanding" vsizetype="Preferred">
        <horstretch>0</horstretch>
        <verstretch>20</verstretch>
       </sizepolicy>
      </property>
      <property name="orientation">
       <enum>Qt::Horizontal</enum>
      </property>
      <widget class="QListView" name="sites_lv">
       <property name="contextMenuPolicy">
        <enum>Qt::ActionsContextMenu</enum>
       </property>
       <property name="selectionMode">
        <enum>QAbstractItemView::ExtendedSelection</enum>
       </property>
      </widget>
      <widget class="QListView" name="proxy_lv">
       <property name="contextMenuPolicy">
        <enum>Qt::ActionsContextMenu</enum>
       </property>
       <property name="selectionMode">
        <enum>QAbstractItemView::ExtendedSelection</enum>
       </property>
      </widget>
     </widget>
    </item>
   </layout>
  </widget>
  <widget class="QToolBar" name="toolBar">
   <property name="windowTitle">
    <string>toolBar</string>
   </property>
   <property name="toolButtonStyle">
    <enum>Qt::ToolButtonTextUnderIcon</enum>
   </property>
   <attribute name="toolBarArea">
    <enum>TopToolBarArea</enum>
   </attribute>
   <attribute name="toolBarBreak">
    <bool>false</bool>
   </attribute>
   <addaction name="actionStart"/>
   <addaction name="separator"/>
   <addaction name="actionSearch"/>
   <addaction name="actionFilter"/>
   <addaction name="actionImport"/>
  </widget>
  <action name="actionStart">
   <property name="icon">
    <iconset>
     <normaloff>control_play_blue.png</normaloff>control_play_blue.png</iconset>
   </property>
   <property name="text">
    <string>Старт</string>
   </property>
   <property name="toolTip">
    <string>Старт</string>
   </property>
  </action>
  <action name="actionSearch">
   <property name="icon">
    <iconset>
     <normaloff>google.png</normaloff>google.png</iconset>
   </property>
   <property name="text">
    <string>Выдача</string>
   </property>
   <property name="toolTip">
    <string>Выдача</string>
   </property>
  </action>
  <action name="actionImport">
   <property name="icon">
    <iconset>
     <normaloff>folder.png</normaloff>folder.png</iconset>
   </property>
   <property name="text">
    <string>Импорт</string>
   </property>
   <property name="toolTip">
    <string>Импорт</string>
   </property>
  </action>
  <action name="actionFilter">
   <property name="icon">
    <iconset>
     <normaloff>filter.png</normaloff>filter.png</iconset>
   </property>
   <property name="text">
    <string>Сортировка</string>
   </property>
   <property name="toolTip">
    <string>Сортировка</string>
   </property>
  </action>
  <action name="actionDelete">
   <property name="icon">
    <iconset>
     <normaloff>delete.png</normaloff>delete.png</iconset>
   </property>
   <property name="text">
    <string>Удалить</string>
   </property>
   <property name="toolTip">
    <string>Удалить</string>
   </property>
  </action>
 </widget>
 <resources/>
 <connections/>
</ui>
